### ISSUE TITLE
Update Page Teaser view

### ISSUE THREAD

**Issue by danahertzberg** [az-digital/az_quickstart#4163]
## Problem/Motivation
On this page, I want it to be easier to use for sighted users and those using screen readers.

### Describe the bug
Aria attributes are not helping as much as they should. It's difficult to know that I can click on an item for more details.

### To Reproduce

1. Visit https://commencement.arizona.edu/ceremonies
2. Inspect to see lack of aria-hidden attribute
3. Hover to see lack of indication that I can click

## Proposed resolution

- The Read More links will be hidden from screen reader view using an ARIA-Hidden attribute.
- We will make the title underlined to visually make users aware that they can also click.

### Expected behavior

1. I want to see an underline of the title when I hover.
2. I want an added aria-hidden attribute to the read more link.
